frontend/src/pages/project/Settings/Danger.vue

```diff
@@ -78,6 +78,19 @@
             </div>
         </div>
 
+        <FormHeading class="text-red-700">Suspend Project</FormHeading>
+        <div class="flex flex-col lg:flex-row max-w-2xl space-y-4">
+            <div class="flex-grow">
+                <div class="max-w-sm pt-2">
+                    Once suspended, your project will not be available until restarted.
```

Comment:
Would like to see some extra clarity over what "Suspend" does here. Think we discussed mentioning of billing here too.

Reply:
We did, but I would like to gate that on having billing enabled